language-support/ts/daml-ledger/index.ts

```diff
@@ -517,12 +536,30 @@ class Ledger {
       } else {
         console.error('Ledger.streamQuery unknown message', json);
       }
-    });
+    };
+    const onClose = ({ code, reason }: {
+        wasClean: boolean; code: number;
+        reason: string; target: WebSocket;
+    }): void => {
```

Comment:
There's no need to mention `wasClean` and `target` if we don't use them.
```suggestion
    const onClose = ({ code, reason }: { code: number; reason: string }): void => {
```